dask_databricks/databrickscluster.py

```diff
@@ -29,6 +29,13 @@ def __init__(
                 "Unable to find expected environment variable SPARK_LOCAL_IP. "
                 "Are you running this on a Databricks driver node?"
             )
+        if os.getenv("MASTER") and "local[" in os.getenv("MASTER"):
+            raise EnvironmentError(
+                "You appear to be running dask-databricks on a "
+                "single-node cluster. Dask requires at least one worker node "
+                "in order to function as expected."
```

Comment:
Could we reword this a little to avoid confusing users. Dask can run just fine on a single node with `LocalCluster`, but in `dask-databricks.DatabricksCluster` we choose not to run any workers on the driver node to match the behaviour or Spark. I don't want folks to see this and thing "oh I can't use Dask on a single-node cluster".

Maybe something like this would be clearer:

```suggestion
                "You appear to be trying to run a multi-node Dask cluster on a single-node Databricks cluster. "
                "Maybe you want `dask.distributed.LocalCluster().get_client()` instead."
```